.kokoro/release/stage.sh

```diff
@@ -27,7 +27,24 @@ MAVEN_SETTINGS_FILE=$(realpath $(dirname "$0")/../../)/settings.xml
 pushd $(dirname "$0")/../../
 
 setup_environment_secrets
-mkdir -p ${HOME}/.gradle
-create_gradle_properties_file "${HOME}/.gradle/gradle.properties"
+create_settings_xml_file "settings.xml"
```

Comment:
This step is required for Maven but not gradle? What exactly are we creating here?

Reply:
create_gradle_properties_file already exists. We're creating the settings to stage the files to sonatype. It's from the monorepo: https://github.com/googleapis/google-cloud-java/blob/8e380cb8857332c44c34bd286297526f7bfb18aa/.kokoro/release/stage.sh#L28